x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -146,21 +146,36 @@ export class ReportListing extends Component<Props, State> {
         field: 'status',
         name: 'Status',
         render: (status: string, record: Job) => {
+          if (status === 'pending') {
+            return <div>pending - waiting for polling worker to claim the job</div>;
```

Comment:
"polling worker" might be a detail users don't need to know.

Thoughts on something like `pending - waiting for job to be processed`.